### PR TITLE
Add a list of third-party blocks

### PR DESCRIPTION
**Pull request by sapegin**
- List of third-party blocks
- Fix title casing
- Update TOC
- Add missed Elm block

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,11 +15,14 @@ Ready to use blocks to configure popular tools like *Babel*, *PostCSS*, *Sass*,
...
-- [Design principles](#design-principles)
+- [Example](#example)
+- [More Examples](#more-example)
+- [Custom Blocks](#custom-blocks)
+- [Available Webpack Blocks](#available-webpack-blocks)
+- [Third-party Blocks](#third-party-blocks)
+- [Design Principles](#design-principles)
 - [FAQ](#faq)
-- [Like what you see?](#like-what-you-see)
+- [Like What You See?](#like-what-you-see)
```
Comment (jvanbruegge) [inline README.md]: I think that's weird, as it is a normal sentence

Comment (sapegin) [inline README.md]: No, it’s a heading, so it should be capitalized as all other headings. Personally I never use title case but I see that @andywer is using it in other documents.

Comment (andywer) [inline README.md]: Mixed feeling. Looks a bit strange indeed, but inconsistency is annoying as well... Not sure, mixed feelings about both our options 😅

PS: Same applies for "Available Webpack Blocks" (strange to see uppercase webpack, esp. if it's not even the first word in a sentence.

**Comment by sapegin** [comment]
The best solution in my opinion is not to use title casing at all ;-) It’s harder to read, it doesn't look good and it’s [hard to do right](http://titlecapitalization.com/).

For example [Google suggests](https://material.io/guidelines/style/writing.html) to use sentence case everywhere in the UI.

**Comment by andywer** [comment]
We can also stick to sentence-style capitalization, I don't mind.

@jvanbruegge @zcei?

**Comment by jvanbruegge** [comment]
I'm in favor of sentence captitalization. Spelling everything uppercase just looks weird

**Comment by sapegin** [comment]
OK, three 👍  for ~~sane~~ sentence casing, I’ll update the PR and send a new one to change the reset of the docs.

**Comment by sapegin** [comment]
Done.

File: README.md [hunk 2]
```diff
@@ -163,16 +166,28 @@ Check out the [sample app](./test-app) to see a webpack config in action or read
...
+- [typescript](./packages/typescript)
 - [uglify](./packages/uglify)
 - [webpack](./packages/webpack)
 
+
+## Third-party blocks
+
+- [webpack-blocks-happypack](https://github.com/diegohaz/webpack-blocks-happypack) — HappyPack
+- [webpack-blocks-purescript](https://github.com/ecliptic/webpack-blocks-purescript) — PureScript
+- [webpack-blocks-server-source-map](https://github.com/diegohaz/webpack-blocks-server-source-map) — source map for server bundle
```
Comment (andywer) [inline README.md]: One idea before I approve: Might make sense to upper-case the first word of the description for a more consistent look ("- Source map for server bundle", "- Vendor bundle")

Comment (sapegin) [inline README.md]: Hm, but it’s not a new sentence…

Comment (andywer) [inline README.md]: Just thought that it might look more consistent, since `HappyPack`, `PureScript`, `TypeScript`, ... all start with upper-case letters.

Comment (sapegin) [inline README.md]: Because they are names ;-)

Comment (andywer) [inline README.md]: Sure, I just mean that it looks a tiny bit odd. But I don't really mind, just wanted to get a 2nd opinion 😉

Comment (sapegin) [inline README.md]: That’s because you used to semirandomly capitalized words in German ;-)

**Pull request merged**